Assets/MRTK/Providers/Oculus/XRSDK/Controllers/OculusXRSDKTouchController.cs

```diff
@@ -108,9 +108,20 @@ protected override bool TryRenderControllerModel(System.Type controllerType, Inp
             }
         }
 
-        private async void TryRenderControllerModelFromOculus()
+        private void TryRenderControllerModelFromOculus()
         {
-            await WaitForOculusVisuals();
+            OculusXRSDKDeviceManager deviceManager = CoreServices.GetInputSystemDataProvider<OculusXRSDKDeviceManager>();
+
+            GameObject platformVisualization = null;
+            if (ControllerHandedness == Handedness.Left)
+            {
+                platformVisualization = deviceManager.leftControllerHelper.gameObject;
+            }
+            if (ControllerHandedness == Handedness.Right)
```

Comment:
nit: could be `else if`, since these cases seem mutually exclusive
```suggestion
            else if (ControllerHandedness == Handedness.Right)
```